File: Iniciante/URI 2234.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var line = lines.shift().split(' ');
var H = parseFloat(line[0]);
var P = parseFloat(line[1]);

var media = H / P;
console.log(media.toFixed(2));

File: Iniciante/URI 2234.clj
(require '[clojure.string :as str])

(def line (str/split (read-line) #" "))
(def H (Float/parseFloat (get line 0)))
(def P (Float/parseFloat (get line 1)))

(def media (/ H P))

(printf "%.2f\n" media)